stl/src/atomic.cpp

```diff
@@ -31,6 +37,7 @@ _CRTIMP2_PURE void __cdecl _Unlock_shared_ptr_spin_lock() { // release previousl
 #else // _M_ARM
     _interlockedbittestandreset(&_Shared_ptr_flag, 0); // reset bit 0
 #endif // _M_ARM
+    __std_atomic_notify_one_direct(const_cast<const long*>(&_Shared_ptr_flag));

```

Comment:
make this notify only called if passed spinlock phase (use more bits of the spinlock here)

Reply:
Resolved by not touching it for now